client/webserver/site/src/js/markets.js

```diff
@@ -349,14 +352,30 @@ export default class MarketsPage extends BasePage {
     const page = this.page
     const market = this.market
     const asset = app.assets[assetID]
-    switch (assetID) {
-      case (market.base.id):
+    const [b, q] = [market.base, market.quote]
+    if (!b) {
```

Comment:
`updateWallet` only updates one wallet. You're now trying to update two at the same time. Return the intended behavior here of updating 1 wallet per call, but continue to hide the box if the asset is undefined.

Move the `loaderMsg` logic elsewhere. Maybe make a separate function `setFormVisibility` that sets the `loaderMsg` and order form visibility and call it when the market is changed.